.buildbot.sh

```diff
@@ -14,6 +16,71 @@ find . -iname "*.c" -o -iname "*.h" -o -iname "*.cpp" -o -iname "*.hpp" | xargs
 
 repodir=$(pwd)
 
+# Sqlite3 splitting initial proof of concept on Linux (Ubuntu 20.04)
+
+# Build custom llvm-13 with dynamic libraries
+wget https://github.com/llvm/llvm-project/releases/download/llvmorg-13.0.0/llvm-project-13.0.0.src.tar.xz
+tar -xf llvm-project-13.0.0.src.tar.xz
+cd llvm-project-13.0.0.src
+mkdir -p build
+cd build
+cmake -DLLVM_ENABLE_PROJECTS="clang;llvm;lld;clang-tools-extra" -DLLVM_LINK_LLVM_DYLIB=ON -G "Unix Makefiles" ../llvm/.
+export CMAKE_BUILD_PARALLEL_LEVEL=32
+cmake --build .
+cmake -DCMAKE_INSTALL_PREFIX=/home/buildbot/llvm-13-with-dynlibs -P cmake_install.cmake
+
+export CUSTOM_CC=$HOME/llvm-13-with-dynlibs
+export LD_LIBRARY_PATH=$CUSTOM_CC/lib
+export CC=$CUSTOM_CC/bin/clang 
+export LLVM_EXTRACT=$CUSTOM_CC/bin/llvm-extract
+export AR=$CUSTOM_CC/bin/llvm-ar 
+export RANLIB=$CUSTOM_CC/bin/llvm-ranlib
+export LLVM_LINK=$CUSTOM_CC/bin/llvm-link
+
+cd $repodir
+# Install gllvm
+go get github.com/SRI-CSL/gllvm/cmd/...
+export GCLANG=$HOME/go/bin/gclang
+export GET_BC=$HOME/go/bin/get-bc
+export LLVM_COMPILER_PATH=$CUSTOM_CC/bin
+
+make CC=$CC CXX=$CUSTOM_CC/bin/clang++ LLVM_CONFIG=$CUSTOM_CC/bin/llvm-config LLVM_LINK=$LLVM_LINK -j4
+export SPLITTER=$repodir/split-llvm-extract
+wget https://www.sqlite.org/src/tarball/sqlite.tar.gz?r=release -O sqlite-latest.tar.gz
+tar -xvf sqlite-latest.tar.gz
+cd sqlite
+mkdir -p build
+cd build
+export CC=$GCLANG
+CC=$CC ../configure
+sed -i '/^CFLAGS =/ s/$/ -fPIC/' Makefile
+make -j32
+# Generate test harness first
+make test -j32
+# Sqlite3 binaries to split
+binaries=( "sqlite3" "lemon" "mkkeywordhash" "sqlite3_analyzer" 
+           "sqltclsh" "fuzzcheck" "sqldiff" "dbhash" "srcck1")
+
+
```

Comment:
Double blank line.

Reply:
Fix in [97830bb](https://github.com/capablevms/llvm-function-split/pull/26/commits/97830bbcac5225b7ae2f2781eeea3cdaa03c7002).